### ISSUE TITLE
Query about TextSnatcher and Tesseract Models

### ISSUE THREAD

**Issue by lemonmeant** [RajSolai/TextSnatcher#51]
Dear Raj Solai,

Hope you're well. First, I've got to say, TextSnatcher is impressive!

I've got a question: I often work with multiple languages, and I'm wondering if there's a way to integrate additional Tesseract language models into TextSnatcher? It would really streamline things for me.

Thanks for your time and for creating such a standout OCR tool.

**Comment by RajSolai** [comment]
Yes tesseract can work with multiple language models at once, soon I will add a multi select box / try to put a auto detect
